package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Do we need this, or can we delete it? The root of the runner shouldn't be a node project

Reply:
My mistake, it's unnecessary, I deleted it